money-api/src/main/java/com/comcast/money/annotations/Timed.java

```diff
@@ -23,14 +23,14 @@
 
 /**
  * Used to indicate that the code path should be traced and executes
- * <p/>
+ * <p>
  * This annotation can be used on any method that you want to trace.  Typically this will be
  * used on the controller to indicate a new request trace, as well as external service methods
- * <p/>
```

Comment:
Shouldn't we at least close the <p> tags we open? I guess it doesn't really matter, but seems like we might as well bring what little balance to the universe we can